flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/MetricMapper.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.metrics.influxdb;
+
+import org.apache.flink.metrics.Counter;
+import org.apache.flink.metrics.Gauge;
+import org.apache.flink.metrics.Histogram;
+import org.apache.flink.metrics.HistogramStatistics;
+import org.apache.flink.metrics.Meter;
+
+import org.influxdb.dto.Point;
+
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+
+class MetricMapper {
+
+	static Point map(MeasurementInfo info, Instant timestamp, Gauge<?> gauge) {
+		Point.Builder builder = builder(info, timestamp);
+		Object value = gauge.getValue();
+		if (value instanceof Number) {
+			builder.addField("value", (Number) value);
+		} else {
+			builder.addField("value", String.valueOf(value));
+		}
+		return builder.build();
+	}
+
+	static Point map(MeasurementInfo info, Instant timestamp, Counter counter) {
+		return builder(info, timestamp)
+			.addField("count", counter.getCount())
+			.build();
+	}
+
+	static Point map(MeasurementInfo info, Instant timestamp, Histogram histogram) {
+		HistogramStatistics statistics = histogram.getStatistics();
+		return builder(info, timestamp)
+			.addField("run-count", histogram.getCount())
```

Comment:
let's keep the names in sync with the StatsD/Slf4j reporter.

Reply:
Personally I'm fine with this, but this would brake compatibility with other [reporter](https://github.com/davidB/metrics-influxdb/blob/master/src/main/java/metrics_influxdb/measurements/MeasurementReporter.java#L129-L139)

Reply:
unless this naming scheme is endorsed/encouraged by influx-db themselves I'd like to make Flink reporters more consistent so it's easier for users to migrate from one reporter to another.